### PR TITLE
Feature/grafana

### PR DESCRIPTION
**Pull request by WinnerOK**

### CODE REVIEW

**Comment by WinnerOK** [comment]
New environment variables:

* `GRAFANA_DB_USER` - username for database user for grafana (will be created if not exists)
* `GRAFANA_DB_PASSWORD` - password for database grafana user
* `GF_SECURITY_ADMIN_PASSWORD` - admin password for Grafana Dashboard (set on initial startup - `admin` if variable is not set)

File: compose/docker-compose-prod.yml [hunk 1]
```diff
@@ -32,6 +32,21 @@ services:
...
+    ports:
+      - "127.0.0.1:3000:3000"
+    environment:
+      GF_SERVER_SERVE_FROM_SUB_PATH: "true"
+      GF_SERVER_ROOT_URL: "%(protocol)s://%(domain)s/dashboard"
+      GF_AUTH_PROXY_ENABLED: "true"
+      GF_AUTH_PROXY_HEADER_NAME: "X-WEBAUTH-USER"
+      GF_AUTH_PROXY_HEADER_PROPERTY: "username"
+      GF_AUTH_PROXY_AUTO_SIGN_UP: "true"
+      GF_AUTH_PROXY_LDAP_SYNC_TTL: "60"
```
Comment (k1rill-fedoseev) [inline compose/docker-compose-prod.yml:46]:
```suggestion
      GF_AUTH_PROXY_LDAP_SYNC_TTL: "60"
      GF_SECURITY_ADMIN_PASSWORD: ${GF_SECURITY_ADMIN_PASSWORD}
```

File: compose/docker-compose.yml [hunk 2]
```diff
@@ -52,6 +52,20 @@ services:
...
+    ports:
+      - "127.0.0.1:3000:3000"
+    environment:
+      GF_SERVER_SERVE_FROM_SUB_PATH: "true"
+      GF_SERVER_ROOT_URL: "%(protocol)s://%(domain)s/dashboard"
+      GF_AUTH_PROXY_ENABLED: "true"
+      GF_AUTH_PROXY_HEADER_NAME: "X-WEBAUTH-USER"
+      GF_AUTH_PROXY_HEADER_PROPERTY: "username"
+      GF_AUTH_PROXY_AUTO_SIGN_UP: "true"
+      GF_AUTH_PROXY_LDAP_SYNC_TTL: "60"
```
Comment (k1rill-fedoseev) [inline compose/docker-compose.yml:66]:
```suggestion
      GF_AUTH_PROXY_LDAP_SYNC_TTL: "60"
      GF_SECURITY_ADMIN_PASSWORD: ${GF_SECURITY_ADMIN_PASSWORD}
```

**Comment by WinnerOK** [comment]
Todo: provision dashboard and data source (https://grafana.com/docs/grafana/latest/administration/provisioning/#using-environment-variables)

**Review by k1rill-fedoseev — APPROVED** [review]

**Pull request merged**